Sources/SwiftSyntax/SyntaxRewriter.swift.gyb

```diff
@@ -198,26 +310,26 @@ public extension SyntaxAnyVisitor {
 % for node in SYNTAX_NODES:
 %   if is_visitable(node):
   mutating func visit(_ node: ${node.name}) -> SyntaxVisitorContinueKind {
-    return visitAny(node)
+    return visitAny(Syntax(node))
```

Comment:
I suppose these can also call `node._syntaxNode`?

Reply:
I thought this cast was zero-cost because we don’t use `node` after the cast. I measured again and was wrong. Thanks for catching this.